src/components/header/Search.tsx

```diff
@@ -47,10 +47,12 @@ export function Search() {
   const clearSearch = useClearSearch();
   const placeholder = useSearchPlaceHolderConfig();
   const autoFocus = useAutoFocusSearchConfig();
-  const { onChange } = useFilter();
+  const { searchResultsCount, searchTerm, onChange } = useFilter();
 
   const input = SearchInputRef?.current;
   const value = input?.value;
+  const statusSearchResults = `${searchResultsCount} result${searchResultsCount > 1 ? 's' : ''} found. Use up and down arrow keys to navigate.`;
+  const statusNoSearchResults = 'No results found';
```

Comment:
I would put these strings in the config object, even if only internal for now, but it will allow to support setting a different language or a string.

Reply:
I moved the strings to `config.ts` and added a `useSearchResultsConfig` function to return  an appropriate string based on search results.  Let me know if it works!

Reply:
Cool. I'll work to make this user configurable in a future commit.